src/device.rs

```diff
@@ -85,6 +85,27 @@ impl UsbDevice {
         res
     }
 
+    pub fn set_configuration_name(&mut self, name: &str) {
+        self.string_pool.remove(&self.string_configuration);
```

Comment:
Error handling is required to avoid panic-ing in library.

Reply:
Fixed!